metabase/metabase
# Inline deprecated driver warning

Epic: https://github.com/metabase/metabase/issues/19624

A new driver warning that is displayed inline in the form.

<img width="1017" alt="Screenshot 2022-01-12 at 15 32 41" src="https://user-images.githubusercontent.com/8542534/149141686-228aac69-774c-4515-b2bb-d0f6c16326df.png">
<img width="817" alt="Screenshot 2022-01-12 at 15 32 50" src="https://user-images.githubusercontent.com/8542534/149141703-6277b8c8-eb82-4558-bc6c-6489a909bdf3.png">
<img width="725" alt="Screenshot 2022-01-12 at 15 36 02" src="https://user-images.githubusercontent.com/8542534/149141710-179d4f58-1003-4b0b-bf7b-a173b565874c.png">
<img width="765" alt="Screenshot 2022-01-12 at 15 36 09" src="https://user-images.githubusercontent.com/8542534/149141717-8d273fa0-b563-4774-a58f-e27ac562e3da.png">

How to test setup changes:
- Run a fresh Metabase instance
- On a database setup step, select `Presto`
- Make sure toggling between the new and the old driver works

How to test admin changes:
- Open Admin > Database > Add
- Select `Presto` driver
- Make sure toggling between the new and the old driver works
